openfga/v1/openfga_service.proto

```diff
@@ -731,6 +745,129 @@ message ListObjectsResponse {
   ];
 }
 
+message ListUsersRequest {
+  string store_id = 1 [
+    json_name = "store_id",
+    (validate.rules).string = {pattern: "^[ABCDEFGHJKMNPQRSTVWXYZ0-9]{26}$"},
+    (grpc.gateway.protoc_gen_openapiv2.options.openapiv2_field) = {example: "\"01YCP46JKYM8FJCQ37NMBYHE5X\""}
+  ];
+
+  string authorization_model_id = 2 [
+    json_name = "authorization_model_id",
+    (validate.rules).string = {
+      pattern: "^[ABCDEFGHJKMNPQRSTVWXYZ0-9]{26}$",
+      ignore_empty: true
+    },
+    (grpc.gateway.protoc_gen_openapiv2.options.openapiv2_field) = {example: "\"01G5JAVJ41T49E9TT3SKVS7X1J\""}
+  ];
+
+  Object object = 3 [
+    json_name = "object",
+    (grpc.gateway.protoc_gen_openapiv2.options.openapiv2_field) = {example: "\"document:example\""},
+    (validate.rules).message.required = true,
+    (google.api.field_behavior) = REQUIRED
+  ];
+
+  string relation = 4 [
+    (validate.rules).string = {pattern: "^[^:#@\\s]{1,50}$"},
+    (google.api.field_behavior) = REQUIRED,
+    (grpc.gateway.protoc_gen_openapiv2.options.openapiv2_field) = {example: "\"reader\""}
+  ];
+
+  repeated ListUsersFilter user_filters = 5 [
```

Comment:
Can we make this type more generic (maybe `UserTypeFilter`) and move it into `openfga.proto`? I see it being used in other APIs across protobuf packages within and even outside of `openfga.v1` . For example,

```
// proto/openfga/v1alpha1/indexer_service.proto
package openfga.v1alpha1

ExpandedReadChanges({
  store_id: string
  authorization_model_id: string
  object_types: []string
  relations: []string
  user_filters: []UserTypeFilter
```